### PR TITLE
keep kombu and pycurl because celery needs them

### PR DESCRIPTION
**Pull request by infinitewarp**
This reverts commit 988dd1d1188cafbba3b8ed33f024c478020c0709.

This should resolve https://github.com/cloudigrade/cloudigrade/issues/353

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -13,3 +13,5 @@ django-rest-polymorphic==0.1.7
 django-celery-beat==1.1.1
 django-health-check==3.6.1
 djoser==1.1.5
+kombu==4.2.1
```
Comment (ilyavw) [inline requirements/base.txt]: I see that kombu also returns, can we restore just pycurl, or must we bring kombu in with it too?

Comment (infinitewarp) [reply requirements/base.txt]: Celery uses Kombu under the covers, which in turn uses pycurl. Celery's dependency chain should automatically pull in Kombu, but pycurl is an optional dependency that you have to explicitly install.

Comment (infinitewarp) [reply requirements/base.txt]: That being said… Celery _should_ automatically get Kombu. I could take it out to let Celery just get whatever version it thinks it needs. What are you thoughts on that, @ilyavw ?

Comment (ilyavw) [reply requirements/base.txt]: I think that would be good, maybe one day it'll pull a version in that doesn't require pycurl.

Comment (ilyavw) [reply requirements/base.txt]: At the same time... we should probably be explicit in our versioning, ¯\\\_(ツ)\_/¯

**Pull request merged**